migrate/live-migration/live-migration-faq.md

```diff
@@ -0,0 +1,85 @@
+---
+title: FAQ
+excerpt: Live-migration Frequently Asked Questions
+products: [cloud]
+keywords: [migration, low-downtime, live-migration, FAQ]
+tags: [migration, logical backup, replication]
+---
+
+# Frequently Asked Questions
+
+This document addresses frequently asked questions and common scenarios you may
+encounter during live migration.
+
+
+## Where can I find logs for processes running during live migration?
+
+Live migration involves several background processes to manage different stages of
+the migration. The logs of these processes can be helpful for troubleshooting
+unexpected behavior. You can find these logs in the `<volume_mount>/logs` directory.
+
+
+## Why does pgcopydb log "no tuple identifier" warning?
```

Comment:
IMO mentioning pgcopydb here is confusing.

```suggestion
## Why does live migration log "no tuple identifier" warning?
```

Reply:
Note: At present, the health checker in live migration will not log "no tuple identifier" to the main logging view. This is because we only treat `ERROR` as the reason for print. I will update the behaviour in live-migration.